lib/verify-stream.js

```diff
@@ -100,7 +101,7 @@ util.inherits(VerifyStream, Stream);
 VerifyStream.prototype.verify = function verify() {
   try {
     var valid = jwsVerify(this.signature.buffer, this.algorithm, this.key.buffer);
-    var obj = jwsDecode(this.signature.buffer, this.encoding);
+    var obj = jwsDecode(this.signature.buffer, {json: this.json, encoding: this.encoding});
```

Comment:
note: `encoding` option was previously not being passed correctly here, which is fine since it was ignored, see https://github.com/auth0/node-jws/pull/86#discussion_r1090150451